### ISSUE TITLE
total demand files contains unneccesary outputs

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1192]
the total demand file contains many variables that are not needed or not interesting.
They were added to the hourly demand files for the pure purpose of creating plots.
They can be removed from the monthly outputs.

List to remove:

Q_gain_sen_base0_kW | Qhs_sen_ahu_MWhyr | Q_gain_lat_peop_MWhyr | Q_loss_sen_ref0_kW | Q_gain_sen_vent_MWhyr | etc. ...

**Comment by gabriel-happle** [comment]
Hi @JIMENOFONSECA I created this issue last week because I noticed that all the variables I added to the building outputs eventually also show up in the `Total_demand.csv` where they are of no use.
I think this is somehow connected also to the interface and the advanced output options.
We can implement it at any stage before the executive course. I don't think there is a rush now. If you agree we can take care of this shortly before the release.
